ex6/index.js

```diff
@@ -0,0 +1,37 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert side A: ");
+const sideA = prompt(">");
+
+console.log("Insert side B: ");
+const sideB = prompt(">");
+
+console.log("Insert side C: ");
+const sideC = prompt(">");
+
+ function verify() {
+    const isSideAValid = sideA < sideB + sideC;
+    const isSideBValid = sideB < sideA + sideC;
+    const isSideCValid = sideC < sideA + sideB;
+
+    if (isSideAValid && isSideBValid && isSideCValid) {
+        return true;
+    } 
+        return false;     
+}
+
+function typeOfTriangle(){
+    if (sideA === sideB && sideB === sideC) {
+        console.log("It's a Equilateral triangle.");
+    }else if (sideA === sideB || sideA === sideC || sideB === sideC) {
+        console.log("It's a Isosceles triangle.");
+    }else if (sideA !== sideB || sideB !==sideC) {
+        console.log("It's a Scalene triangle.");
+    }
```

Comment:
Having the extra else at the end is unnecessary and can lead to incorrect classification. We can simplify this to something like below:

```js
    if (a === b && b === c) {
        console.log("It's an Equilateral triangle.");
    } else if (a === b || a === c || b === c) {
        console.log("It's an Isosceles triangle.");
    } else {
        console.log("It's a Scalene triangle.");
    }
```

Reply:
I create a new const? Like const a, const b, or I use sideA ?

Reply:
Nope, my code was just an example. You can use the names you have in your code.